### PR TITLE
Essentials Review Comments, Added logo.png and made other changes to documents

### PR DESCRIPTION
**Pull request by sharma-tapas**
1. Added px-logo.png
2. Removed the drop down for KVDB
3. Added link to Essentials pre-requisites in the README.md

Signed-off-by: Tapas Sharma <tapas@portworx.com>

### CODE REVIEW

File: charts/portworx-essentials/v1.0.0/values.yaml [hunk 1]
```diff
@@ -26,7 +26,7 @@ envVars: none                         # NOTE: This is a ";" seperated list of en
 stork: true                           # Use Stork https://docs.portworx.com/scheduler/kubernetes/stork.html for hyperconvergence.
 storkVersion: 2.4.2
 
-lighthouse: true
+lighthouse: false
```
Comment (harsh-px) [inline charts/portworx-essentials/v1.0.0/values.yaml:29]: remove portworx-lighthouse.yaml completely

File: charts/portworx-essentials/v1.0.0/README.md [hunk 2]
```diff
@@ -1,9 +1,30 @@
...
+
+### Hardware & Software
+	
+|Resource|Requirements|
+|--------|-----------|
+|CPU|4 cores|
+|RAM|4GB|
+|Disk (/var)| 2GB free|
+|Backing drive|8GB (minimum required) 128 GB (minimum recommended)|
+|Storage drives| Storage drives must be unmounted block storage: raw disks, drive partitions, LVM, or cloud block storage.|
```
Comment (harsh-px) [inline charts/portworx-essentials/v1.0.0/README.md:18]: Add a note here that at-least one node in the cluster needs to have these storage drives

File: charts/portworx-essentials/v1.0.0/values.yaml (hunk 1 above)
Comment (sharma-tapas) [reply charts/portworx-essentials/v1.0.0/values.yaml:29]: done.

File: charts/portworx-essentials/v1.0.0/README.md (hunk 2 above)
Comment (sharma-tapas) [reply charts/portworx-essentials/v1.0.0/README.md]: added a minimum section.

**Pull request merged**